test/gurka/test_instructions_roundabout_multicue.cc

```diff
@@ -0,0 +1,198 @@
+           {"maxspeed", "50 mph"},
+           {"name", ""}}},
+         {"CD", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "1st Avenue"}}},
+         {"EF",
+          {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "Main Street"}, {"ref", "A 1"}}},
+         {"HI", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", ""}}},
+         {"FJLNF",
+          {{"highway", "primary"},
+           {"oneway", "yes"},
+           {"junction", "roundabout"},
+           {"maxspeed", "50 mph"},
+           {"name", ""}}},
+         {"JK", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "2nd Avenue"}}},
+         {"LM",
+          {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "Main Street"}, {"ref", "A 1"}}},
+         {"NO", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", ""}}},
+         {"PM", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "3rd Avenue"}}}};
+
+    const auto layout =
+        gurka::detail::map_to_coordinates(ascii_map, gridsize_metres, {5.1079374, 52.0887174});
+    map = gurka::buildtiles(layout, ways, {}, {}, "test/data/gurka_roundabout_multicue",
+                            {{"mjolnir.admin",
+                              {VALHALLA_SOURCE_DIR "test/data/netherlands_admin.sqlite"}}});
+  }
+};
+
+gurka::map InstructionsRoundaboutMulticue::map = {};
+
+/*************************************************************/
+
+TEST_F(InstructionsRoundaboutMulticue, StartThenEnterRoundaboutMulticueOrdinalOnly) {
+  auto result = gurka::route(map, "A", "I", "auto");
+  gurka::assert::raw::expect_maneuvers(result, {DirectionsLeg_Maneuver_Type_kStart,
+                                                DirectionsLeg_Maneuver_Type_kRoundaboutEnter,
+                                                DirectionsLeg_Maneuver_Type_kRoundaboutExit,
+                                                DirectionsLeg_Maneuver_Type_kDestination});
+  int maneuver_index = 0;
+
+  // Verify the start then enter roundabout multicue
+  gurka::assert::raw::expect_instructions_at_maneuver_index(
+      result, maneuver_index, "Drive east on Main Street/A 1.", "",
+      "Drive east on Main Street, A 1. Then Enter the roundabout and take the 3rd exit.",
+      "Continue for 100 meters.");
+
+  // Verify the enter_roundabout instructions
+  gurka::assert::raw::
+      expect_instructions_at_maneuver_index(result, ++maneuver_index,
+                                            "Enter the roundabout and take the 3rd exit.",
+                                            "Enter the roundabout and take the 3rd exit.",
+                                            "Enter the roundabout and take the 3rd exit.", "");
+
+  // Verify the exit_roundabout instructions
+  gurka::assert::raw::expect_instructions_at_maneuver_index(
+      result, ++maneuver_index, "Exit the roundabout.", "",
+      "Exit the roundabout. Then You will arrive at your destination.", "Continue for 200 meters.");
```

Comment:
Looks like we do a simple concatenation of the text instruction with the `Then` phrase. We should probably look into make the first letter lower-case. Need not be done now though.

Reply:
since it is for voice and not display - did not think it was worth it. I do not hear any issues with using it this way

Reply:
Aren't the text instructions shown somewhere? I guess we can revisit it later if someone reports an issue.